firebase-firestore/src/testUtil/java/com/google/firebase/firestore/testutil/TestUtil.java

```diff
@@ -255,6 +258,27 @@ public static FieldFilter filter(String key, String operator, Object value) {
     return FieldFilter.create(field(key), operatorFromString(operator), wrap(value));
   }
 
+  public static CompositeFilter andFilter(List<Filter> filters) {
```

Comment:
Should we add an "s" to all these methods: `andFIlters()` ?

Reply:
Done